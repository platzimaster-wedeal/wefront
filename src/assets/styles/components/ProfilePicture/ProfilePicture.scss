@import '../../partials/colors.scss';
@import '../../partials/fonts.scss';
@import '../../FontSize.scss';


.image {
    width: 60px;
    height: 60px;
    max-width: 60px;
    max-height: 60px;
    border-radius: 50%;
    object-fit: cover;
    overflow: hidden;

}
.image__isOnline {
    border: 4px solid $color-primary;

}
.image__isOffline {
    border: 4px solid $color-black-60;
}