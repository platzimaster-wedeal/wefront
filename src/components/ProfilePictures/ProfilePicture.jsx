import React from "react";
import "../../assets/styles/components/ProfilePicture/ProfilePicture.scss";

const ProfilePicture = ({ source, alt, active }) => {
 return active ? (
  <img className="image image__isOnline" src={source} alt={alt} />
 ) : (
  <img className="image image__isOffline" src={source} alt={alt} />
 );
};

export default ProfilePicture;
