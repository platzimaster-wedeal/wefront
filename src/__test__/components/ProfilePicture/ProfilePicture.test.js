import React from "react";
import { mount } from "enzyme";
import { create } from "react-test-renderer";
import ProfilePicture from "../../../components/ProfilePictures/ProfilePicture";

describe("<ProfilePicture/>", () => {
 test("Should render the ProfilePicture component", () => {
  const profilePicture = mount(<ProfilePicture />);
  expect(profilePicture.length).toEqual(1);
 });
 test("ProfilePicture Snapshot", () => {
  const profilePicture = create(<ProfilePicture />);
  expect(profilePicture.toJSON()).toMatchSnapshot();
 });
});
