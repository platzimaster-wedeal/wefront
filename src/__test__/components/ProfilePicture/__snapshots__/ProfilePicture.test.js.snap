// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfilePicture/> ProfilePicture Snapshot 1`] = `
<img
  className="image image__isOffline undefined"
/>
`;
